src/main/java/com/jamierf/dropwizard/logging/gelf/GelfAppenderFactory.java

```diff
@@ -66,6 +84,30 @@ public void setServer(HostAndPort server) {
         this.server = server;
     }
 
+    @JsonProperty
+    public void setDefaultFields(Map<String, String> defaultFields)
+    {
```

Comment:
Can you please format this to match the code style used in the rest of the project?